components/time/src/zone/mod.rs

```diff
@@ -203,19 +225,29 @@ pub(crate) mod ule {
     #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
     #[non_exhaustive]
     pub enum TimeZoneVariant {
-        /// The variant corresponding to `"standard"` in CLDR.
+        /// Indicates the period in a time zone with a lower UTC offset than `Daylight`.
```

Comment:
I don't like defining standard in terms of daylight, because most zones don't have a daylight variant...